[Bug Report] missing js-beautify vuetify-storybook

### Environment
**Browsers:** Chromium 80.0.3987.87
**OS:** Linux

### Steps to reproduce
* Create new vue project
* `vue add vuetify`
* `vue add vuetify-storybook`
* yarn run serve:storybook

### Expected Behavior
Storybook page

### Actual Behavior
```
ERROR in ./.storybook/addon-show-vue-markup/decorator.js
Module not found: Error: Can't resolve 'js-beautify' in '/Users/renato.vicente/Desktop/projetos/Morpheus.Client/.storybook/addon-show-vue-markup'
 @ ./.storybook/addon-show-vue-markup/decorator.js 3:0-35 12:14-22
 @ ./.storybook/addon-show-vue-markup/index.js
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js ./node_modules/webpack-hot-middleware/client.js?reload=true&quiet=true
```

### Reproduction Link

--- a/packages/vue-cli-plugin-vuetify-storybook/generator/index.js
+++ b/packages/vue-cli-plugin-vuetify-storybook/generator/index.js
@@ -6,6 +6,7 @@ module.exports = api => {
   api.extendPackage({
     scripts: {
       'serve:storybook': 'start-storybook -p 6006',
+      "build:storybook": "build-storybook",
     },
     devDependencies: {
       '@babel/preset-react': '^7.6.3',
@@ -19,7 +20,8 @@ module.exports = api => {
       '@storybook/vue': '^5.2',
       'babel-preset-vue': '^2.0.2',
       'prism-react-renderer': '^0.1.7',
-      prismjs: '^1.17.1',
+      'js-beautify': '^1.11.0',
+      'prismjs': '^1.17.1',
       'vue-storybook': '^1.1.0',
     },
   })

